client/src/modules/templates/modals/cronEmailModal.html

```diff
@@ -0,0 +1,19 @@
+<div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static"><b><span translate> CRON.EMAIL_PUSHING_SYSTEM</span></b></li>
+    </ol>
+  </div>
+
+<div class="modal-body" style="max-height: 500px; overflow-y:scroll">
```

Comment:
You can use `overflow:auto;` which has a sweet behavior than `overflow-y:scroll` in my opinion